src/pymodaq/daq_utils/h5utils.py

```diff
@@ -0,0 +1,218 @@
+# Standard imports
```

Comment:
the methods in this new module have been moved into the browsing module (of data_management branch). I would rather not modify too much this version compared to the modification I did in the data_management branch. But maybe I will regroup this kind of methods into a h5modules/h5utils module also in data_management. So overall it is a good idea, I will transpose it into the v4 branch! 

Did you check that imports modification of these methods has been properly done in modules requiring them? I'm not sure the existing tests extend to this kind of testing...

Reply:
Yes, I ran a whole project check on this modification and it worked fine,

No problem at run time neither.